src/test/java/org/folio/rest/impl/ApiTestBase.java

```diff
@@ -171,6 +171,7 @@ public JsonObject getMockAsJson(String fullPath) {
     try {
       return new JsonObject(getMockData(fullPath));
     } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
please, use the logger instead of printStackTrace

Reply:
whoops!  Good catch